Cargo.toml

```diff
@@ -8,6 +8,11 @@ homepage = "https://www.neon-bindings.com"
 repository = "https://github.com/neon-bindings/neon"
 license = "MIT/Apache-2.0"
 exclude = ["neon.jpg"]
+build = "build.rs"
+
+# This name is arbitrary, but allows us to re-export the location of node.lib
+# to dependent packages so they can link on Windows.
+links = "neon-sys"
```

Comment:
I'll tweak the comment but separately I've filed #169 to rename `neon-sys`. Thanks for calling this out.